common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -1,24 +1,60 @@
 ---
```

Comment:
@cptvitamin I don't think this could affect the work in https://github.com/edx/edx-platform/pull/7564 but still I would try to test this with that one.